great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -158,12 +154,20 @@ def _init_rules(
                     )
 
             # Instantiate builder attributes
-            domain_builder = self._init_domain_builder(rule_config, data_context)
+            domain_builder = self._init_domain_builder(
+                domain_builder_config=rule_config["domain_builder"],
+                data_context=data_context,
+            )
             parameter_builders = self._init_parameter_builders(
-                rule_config, data_context
+                parameter_builder_configs=rule_config["parameter_builders"],
```

Comment:
We aren’t guaranteed a parameter builder key so please ensure you use get here (and use a list in the case of a miss)

Reply:
@cdkini Good catch -- fixed to use `get()`.  Thanks!